src/Raven.Server/Documents/PeriodicBackup/Azure/RavenAzureClient.cs

```diff
@@ -32,16 +32,25 @@ public class RavenAzureClient : RavenStorageClient
         private readonly byte[] _accountKey;
         private readonly string _containerName;
         private readonly string _serverUrlForContainer;
-        private readonly bool _isTest;
         private const string AzureStorageVersion = "2017-04-17";
         private const int MaxUploadPutBlobInBytes = 256 * 1024 * 1024; // 256MB
         private const int OnePutBlockSizeLimitInBytes = 100 * 1024 * 1024; // 100MB
         private const long TotalBlocksSizeLimitInBytes = 475L * 1024 * 1024 * 1024 * 1024L / 100; // 4.75TB
         private readonly Logger _logger;
+        public static bool TestMode;
 
         public RavenAzureClient(AzureSettings azureSettings, Progress progress = null, Logger logger = null, CancellationToken? cancellationToken = null, bool isTest = false)
```

Comment:
You can remove isTest and use the static TestMode.

Reply:
Done